acquisitions/src/main/resources/thunderjet/mod-finance/features/allowable-encumbrance-and-expenditure-restrictions.feature

```diff
@@ -28,7 +28,7 @@ Feature: Test allowable encumbrance and expenditure restrictions
     * def poLineId = call uuid
     * def encumbranceId = call uuid
 
-    * call createFund { 'id': '#(fundId)', 'ledgerId': #(ledgerId) }
+    * def v = call createFund { 'id': '#(fundId)', 'ledgerId': '#(ledgerId)' }
```

Comment:
just for my understanding, why we need to assing result to some variable here?

Reply:
I wrote based on other PRs, it is working without it. That will be deleted

Reply:
we have dozens of similar places, thinking that we need to delete here a well to make code more clean

Reply:
The places introduced in my pr definitely helpful for the Performance testing. I tested with deleted this other similar places, tests failed because of the calling createFund.feature two times. I am not sure why outcome is different but this places helping performance testing working correctly.
![image](https://github.com/folio-org/folio-integration-tests/assets/113523904/fc17ab0b-287d-4f54-948d-747178f8fd6f)